oval/util_test.go

```diff
@@ -2364,6 +2364,68 @@ func TestIsOvalDefAffected(t *testing.T) {
 			fixState:    "Affected",
 			fixedIn:     "",
 		},
+		{
+			in: in{
+				family:  constant.SUSEEnterpriseServer,
+				release: "12.3",
+				def: ovalmodels.Definition{
+					AffectedPacks: []ovalmodels.Package{
+						{
+							Name:    "kernel-default",
+							Version: "0:4.4.140-96.98",
+						},
+					},
+				},
+				req: request{
+					packName:       "kernel-default",
+					versionRelease: "0:4.4.140-96.97",
+					arch:           "x86_64",
+				},
+			},
+			affected: true,
+			fixedIn:  "0:4.4.140-96.98",
+		},
+		{
+			in: in{
+				family:  constant.SUSEEnterpriseServer,
+				release: "12.3",
+				def: ovalmodels.Definition{
+					AffectedPacks: []ovalmodels.Package{
+						{
+							Name:    "kernel-default",
+							Version: "0:4.4.140-96.97.TDC.1",
+						},
+					},
+				},
+				req: request{
+					packName:       "kernel-default",
+					versionRelease: "0:4.4.140-96.97",
+					arch:           "x86_64",
+				},
+			},
+			affected: false,
+		},
```

Comment:
How about inversed pattern which is the very case of the corresponding issue? i.e.

```
		{
			in: in{
				family:  constant.SUSEEnterpriseServer,
				release: "12.3",
				def: ovalmodels.Definition{
					AffectedPacks: []ovalmodels.Package{
						{
							Name:    "kernel-default",
							Version: "0:4.4.180-94.156.1",
						},
					},
				},
				req: request{
					packName:       "kernel-default",
					versionRelease: "4.4.140-96.126.TDC.1",
					arch:           "x86_64",
				},
			},
			affected: false,
		},
```

Reply:
added
https://github.com/future-architect/vuls/compare/d942d938a2c182362eb516596dd2a83451cb13a2..eb9de339d3db209fd6dff8f44a00da9020ca4187